src/runtime/virtcontainers/qemu_arch_base.go

```diff
@@ -813,3 +830,9 @@ func (q *qemuArchBase) getPFlash() ([]string, error) {
 func (q *qemuArchBase) setPFlash(p []string) {
 	q.PFlash = p
 }
+
+// append protection device
+func (q *qemuArchBase) appendProtectionDevice(devices []govmmQemu.Device, firmware string) ([]govmmQemu.Device, string, error) {
+	virtLog.Warnf("Confidential Computing has not been implemented for this architecture")
```

Comment:
Could we put the architecture name in the log, so that when we receive such a log, we know what architecture we are been asked to support ;-)